src/styles/global.scss

```diff
@@ -14,6 +14,86 @@
   }
 }
 
+html.is-extension {
+  width: $extension-width;
+  height: $extension-height;
+
+  // fixes issue with extension not rendering properly
+  ion-tabs,
+  .tabs-inner,
+  ion-router-outlet,
+  .ion-page {
+    contain: layout style !important;
+  }
+
+  body {
+    position: unset;
+  }
+}
+
+// When scanner is active we need to hide everything else
+// because the scanner will show-up behind the webview
+body.scanner-active {
+  background-color: transparent;
+
+  --background: transparent;
+  --ion-background-color: transparent;
+  --screen-bg-color: transparent;
+
+  .app-wrapper {
+    --screen-bg-color: transparent;
+
+    background: transparent;
+  }
+
+  .ionic-wrapper {
+    background-color: transparent;
+  }
+
+  #app {
+    background: transparent;
+    position: absolute;
+    z-index: 2;
+    width: 100%;
+  }
+
+  ion-content,
+  .ion-padding {
+    --background: transparent;
+
+    background: transparent;
+  }
+}
+
+.account-details .ion-padding,
+.account-details-names .ion-padding {
+  background-color: $color-bg-4;
+}
```

Comment:
This are definitely not a global things.

Reply:
Yes, and this was solved using `:deep`.
It was solved using `:deep` because the components we want this to affect are rendered using `<IonRouterOutlet/>` and we cannot pass props through the `<IonRouterOutlet/>`

This is an example were I would love being able to use `:deep` and not have to create an extra composable or anything hacky to solve this